docs/source/user-guide/getting-started.rst

```diff
@@ -2,306 +2,141 @@
-
-#. Verify that the snakes environment has been added and is active:
-
-   .. code::
-
-      conda info --envs
-
-   Conda displays the list of all environments with an asterisk (*)
-   after the name of the active environment:
-
-   .. code::
+.. tip::
+   The active environment is the one with an asterisk (*).
 
-     # conda environments:
-     #
-     base                     /home/username/anaconda3
-     snakes                *  /home/username/anaconda3/envs/snakes
-     snowflakes               /home/username/anaconda3/envs/snowflakes
+To change your current environment back to the default ``base``::
 
-   The active environment is also displayed in front of your prompt in
-   (parentheses) or [brackets] like this:
+   conda activate
 
-   .. code::
+.. tip::
+    When the environment is deactivated, its name is no longer shown in your prompt, and the asterisk (*) returns to ``base``. To verify, you can repeat the  ``conda info --envs`` command.
 
-     (snakes) $
+Installing packages
+===================
 
-#. Verify which version of Python is in your current
-   environment:
+You can also install packages into a previously created environment. To do this, you first need to activate the environment. This changes the environment shown in your shell from ``(base)`` to the name of the environment::
 
-   .. code::
+   conda activate myenvironment
+   conda install matplotlib
 
-      python --version
+For more information on searching for and installing packages, see :doc:`Managing packages <tasks/manage-pkgs>`.
 
-#. Deactivate the snakes environment and return to base environment:
-   ``conda activate``
+Specifying channels
+===================
 
-   .. note::
-      For versions prior to conda 4.6, use:
+Channels are locations (on your own computer or elsewhere on the Internet) where packages are stored. By default, conda searches for packages in its :ref:`default channels <default-channels>`.
 
-        * Windows:  ``activate``
-        * macOS, Linux: ``source activate``
+If a package you want is located in another channel, such as conda-forge, you can manually specify the channel when installing the package::
 
+   conda activate myenvironment
+   conda install conda-forge::numpy
 
-.. _managing-pkgs:
+You can also override the default channels in your `.condarc` file. For a direct example, see :ref:`Channel locations (channels) <config-channels>` or read the entire :doc:`Using the .condarc conda configuration file <configuration/use-condarc>`.
 
-Managing packages
-=================
+.. tip::
 
-In this section, you check which packages you have installed,
-check which are available and look for a specific package and
-install it.
+   Find more packages and channels by searching `Anaconda.org <https://www.anaconda.org>`_. To learn more about searching for packages and channels on Anaconda.org, see `Anaconda.org's documentation <https://docs.anaconda.com/free/anacondaorg/user-guide/>`_.
```

Comment:
I'm not a fan of how repetitive this is, but wanted to convey that the Anaconda.org documentation is outside the conda documentation.